ios/brave-ios/Sources/BraveWallet/Crypto/Accounts/Activity/AccountActivityView.swift

```diff
@@ -41,10 +41,12 @@ struct AccountActivityView: View {
             } label: {
               Label(Strings.Wallet.editButtonTitle, braveSystemImage: "leo.edit.pencil")
             }
-            Button {
-              isPresentingExportAccount = true
-            } label: {
-              Label(Strings.Wallet.exportButtonTitle, braveSystemImage: "leo.key")
+            if store.account.coin != .btc {
```

Comment:
maybe .zec here also?

Reply:
Alright, I think now that handling also .zec is not what we need

Reply:
yep, you are right.